madmom/ml/nn/layers.py

```diff
@@ -344,6 +344,144 @@ def activate(self, data):
         return out
 
 
+class GRUCell(object):
+    """
+    Cell as used by GRU layers proposed in [1].
+
+    Parameters
+    ----------
+    weights_in_to_hidden_update : numpy array, shape ()
+        Weights.
+    weights_in_to_hidden_update : numpy array, shape ()
```

Comment:
I prefer a simple `weights` (in line with all other layers and gates) as the parameter name and a more descriptive description.